assets/misspellings.csv

```diff
@@ -0,0 +1,21 @@
+acounts,accounts
```

Comment:
Should we put the file under `build` directory?

Reply:
It's basically a config file, could instead put it in root or [`.config`](https://github.com/pi0/config-dir), but the number of tools supporting the latter is slim.

Reply:
I've moved it to `tools` now. At least it can't get confused with build assets now.